packages/babel-plugin-extract-export-names/index.js

```diff
@@ -38,16 +42,14 @@ class BabelPluginExtractExportNames {
       }
 
       const handleSpecifiers = node => {
-        const {specifiers} = node
-
-        if (!specifiers) {
-          return
-        }
+        /* istanbul ignore next - seems to always exist, maybe for older Babel? */
+        const specifiers = node.specifiers || []
```

Comment:
Since specifiers is the only part accessed would it make sense to use:

```js
const handleSpecifiers = ({ specifiers = [] }) =>
```
?

Reply:
I’d rather remove the default in a major, because I can’t seem to cover it